SIL.Scripture/VerseRef.cs

```diff
@@ -808,6 +813,11 @@ public bool NextChapter(BookSet present, bool skipExcluded = false)
 			return true;
 		}
 
+		public bool NextChapter(BookSet present)
+		{
+			return NextChapter(present);
```

Comment:
Won't that cause an infinite loop? I guess this should probably be:

```suggestion
			return NextChapter(present, false);
```